test/e2e-go/cli/goal/expect/goalAppAccountAddressTest.exp

```diff
@@ -148,7 +148,7 @@ proc goalAppAccountAddress { TEST_ALGO_DIR TEST_DATA_DIR} {
 	--app-account $ACCOUNT_4_ADDRESS
     expect {
         timeout { puts timeout; ::AlgorandGoal::Abort  "\n Failed to see expected output" }
-        "*Couldn't broadcast tx with algod: HTTP 400 Bad Request: TransactionPool.Remember: transaction*invalid Accounts index 4*" {
+        "*Couldn't broadcast tx with algod: HTTP 400 Bad Request:*TransactionPool.Remember: transaction*invalid Accounts index 4*" {
```

Comment:
New v2 endpoints prefix the error with `"message:"` - match everything else

Reply:
Reverted this change in [bb260ab](https://github.com/algorand/go-algorand/pull/4684/commits/bb260ab860b380cbb64a62806315f2343a549884)